### ISSUE TITLE
$algoliaSettings['attributesForFaceting'] setting is being ignored.

### ISSUE THREAD

**Issue by mspivak** [algolia/algoliasearch-laravel#51]
Hi there,

Each time I run `Product::reindex();` the `attributesForFaceting` setting gets cleared on my dashboard and I must login and update it manually. Am I doing it wrong?

```
class Product extends \ResourceModel {
	use AlgoliaEloquentTrait;
	public static $perEnvironment = true;
	public static $autoIndex = true;
	public $algoliaSettings = [
		'attributesForFaceting' => [ 'category_id', 'vendor_id', 'unit_price', 'brand' ],
	];
}
```

Thanks in advance!

**Comment by Nikita240** [comment]
+1 same issue

**Comment by Nikita240** [comment]
Turns out this is intended behaviour.

See the part in blue:

![image](https://cloud.githubusercontent.com/assets/6479817/12802842/91ace756-ca9a-11e5-8aa1-f74f13461367.png)

**Comment by maxiloc** [comment]
Yes you need to do setSettings after the full reindex. Let keep this open as we might want to handle it in the reindex method

**Comment by mspivak** [comment]
You're right @maxiloc. Doing `setSettings` after `reindex` fixes the thing. I would suggest adding it by default, since one wouldn't expect that reindexing resets the index settings!

**Comment by JanPetr** [comment]
Fixed by #64. Feature will be part of next release.

**Comment by GiampaoloFalqui** [comment]
This issue presents itself also on [algoliasearch-laravel4](https://github.com/algolia/algoliasearch-laravel-4) branch - tested now.
